src/service/poc_mobile.proto

```diff
@@ -80,6 +83,41 @@ message subscriber_location_ingest_report_v1 {
   subscriber_location_req_v1 report = 2;
 }
 
+enum signal_level {
+  NO = 0;
+  LOW = 1;
+  MEDIUM = 2;
+  HIGH = 3;
+}
+
+// Radio signal level and power in the h3 hex.
+message radio_hex_signal_level {
+  uint64 hex_id = 1;
```

Comment:
what is `hex_id` an h3 hex? if so that's called `location`?

Reply:
It is an h3 hex. I see other places use h3 hexes via a string called location, so I will change to that